reana_commons/utils.py

```diff
@@ -284,10 +284,10 @@ def get_disk_usage(
 
     :return: List of dicts with file name and size.
     """
-    reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
-    if not reana_fs.exists(directory):
+    # reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
+    if not os.path.exists(directory):
         raise REANAMissingWorkspaceError("Directory does not exist.")
-    absolute_path = reana_fs.getospath(directory)
+    absolute_path = bytes(directory, "utf-8")
```

Comment:
I think instead encoding it we could remove decoding part [here](https://github.com/reanahub/reana-commons/blob/master/reana_commons/utils.py#L258), since the directory should be the full path already